packages/docs/docusaurus.config.js

```diff
@@ -145,11 +165,11 @@ const config = {
           from: '/react-jsonschema-form/docs/',
           to: '/',
         },
-        // We will need to tune `contextualSearch` and `searchParameters` to handle search for versioned docs
+        // We may need to tune `contextualSearch` and `searchParameters` to handle search for versioned docs
         // Optional: see doc section -- https://docusaurus.io/docs/search#contextual-search
         contextualSearch: true,
         // Optional: Algolia search parameters
-        searchParameters: {},
+        // searchParameters: {},
```

Comment:
What about this change? Maybe document why it is currently commented out

Reply:
We're not currently using it, and I'm not sure if an empty value could interfere with whatever the default behavior might be for versioned search. It probably doesn't make a difference.